themes/aura/snippets/slider.liquid

```diff
@@ -29,6 +29,8 @@
           {{ block.youcan_attributes }}
         >
           <img
+            width="1440"
```

Comment:
The issue still isn't fixed with this method. (`1440 x 610` are the default sizes for the hero image so the browser can **identify** it)

Reply:
the issue will not appear since you have defined width and height attributes but since there is some CSS that overrides those new values nvm you can set anything